### DISCUSSION TITLE
Conditional Vercel Build by committed files

### DISCUSSION THREAD

**Discussion by theodorusclarence** [vercel/community#128] [Help] [answered]
Is there any way to constrain the build according to file changes?

The use case is:

I only want to build storybook deployment when the changes is in `'../src/**/*.stories.mdx'` or `'../src/**/*.stories.@(js|jsx|ts|tsx)'`

Thanks!

**Comment by Pieparker** [comment, marked as answer]
Hey there! I won't be able to write it for you but [this support article](https://vercel.com/support/articles/how-do-i-use-the-ignored-build-step-field-on-vercel#with-folders-and-workspaces) will show you how the ignore build step can ensure builds only occur when particular files/types change.
